subprojects/docs/src/docs/userguide/working_with_files.adoc

```diff
@@ -860,15 +860,16 @@ The filename extension for the archive. By default, this is set based on the arc
 <<#sub:sharing_copy_specs,As described earlier>>, you can use the link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:copySpec(org.gradle.api.Action)[Project.copySpec(org.gradle.api.Action)] method to share content between archives.
 
 [[sec:reproducible_archives]]
-=== Reproducible archives
+[[sec:reproducible_builds]]
+=== Reproducible builds
 
 Sometimes it's desirable to recreate archives exactly the same, byte for byte, on different machines. You want to be sure that building an artifact from source code produces the same result no matter when and where it is built. This is necessary for projects like https://reproducible-builds.org/[reproducible-builds.org].
 
 Reproducing the same byte-for-byte archive poses some challenges since the order of the files in an archive is influenced by the underlying file system. Each time a ZIP, TAR, JAR, WAR or EAR is built from source, the order of the files inside the archive may change. Files that only have a different timestamp also causes differences in archives from build to build. All link:{groovyDslPath}/org.gradle.api.tasks.bundling.AbstractArchiveTask.html[AbstractArchiveTask] (e.g. Jar, Zip) tasks shipped with Gradle include support for producing reproducible archives.
 
 For example, to make a `Zip` task reproducible you need to set link:{groovyDslPath}/org.gradle.api.tasks.bundling.Zip.html#org.gradle.api.tasks.bundling.Zip:reproducibleFileOrder[Zip.isReproducibleFileOrder()] to `true` and link:{groovyDslPath}/org.gradle.api.tasks.bundling.Zip.html#org.gradle.api.tasks.bundling.Zip:preserveFileTimestamps[Zip.isPreserveFileTimestamps()] to `false`. In order to make all archive tasks in your build reproducible, consider adding the following configuration to your build file:
 
-.Activating reproducible archives
+.Activating reproducible builds
```

Comment:
I think reproducible builds is a bigger step than just reproducible archives.  These flags make it so if you take the same inputs files, you'll get a byte-for-byte equivalent archive.

To me, a reproducible build is a build that given the same "source files" will produce byte-for-byte artifacts (which are probably archives).  Just enabling these flags will not get a reproducible build if you have non-repeatable inputs.  For example, if you generate a resource file that you bundle in the jar with a timestamp, you won't get a reproducible build, but you might get a reproducible archive. 

I would leave this title as it is since it might be misleading to say this enables reproducible builds.

Reply:
Your detailed arguments make sense. Thus, I reverted this line change.